frog/imports/client/Wiki/index.js

```diff
@@ -223,7 +225,7 @@ class WikiComp extends Component<WikiCompPropsT, WikiCompStateT> {
     if (!currentPageObj) {
       if (!fullPageObj.noNewInstances) {
         this.initialLoad = true;
-        const instanceName = Meteor.user().username;
+        const instanceName = this.state.user.username;
```

Comment:
why this change?

Reply:
In order to improve the performance, we re-use the value instead of calling `Meteor.user()` again.